### PR TITLE
Chore: Improve test for livechat 

### PR DESCRIPTION
**Pull request by weslley543**
- Improve test for livechat

### CODE REVIEW

File: apps/meteor/tests/e2e/omnichannel-livechat.spec.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { faker } from '@faker-js/faker';
+import type { Page } from '@playwright/test';
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/omnichannel-livechat.spec.ts:2]:
```suggestion
import { Page } from '@playwright/test';
```

File: apps/meteor/tests/e2e/omnichannel-livechat.spec.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+				await expect(page.locator('div >>text="this_a_test_message_from_user"')).toBeVisible();
+			});
+
+			test('expect after user close live chat screen dont show messages', async ({ page }) => {
+				await poLiveChat.btnOpenLiveChat('R').click();
+				await expect(page.locator('[contenteditable="true"]')).not.toBeVisible();
+			});
+		});
+	});
+	test.use({ storageState: 'user1-session.json' });
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/omnichannel-livechat.spec.ts:55]: Create a describe block and put this test.use inside

File: apps/meteor/tests/e2e/page-objects/omnichannel-livechat.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+	get onlineAgentMessage(): Locator {
+		return this.page.locator('[contenteditable="true"]');
+	}
+
+	get btnSendMessageToOnlineAgent(): Locator {
+		return this.page.locator('footer div div div:nth-child(3) button');
+	}
+
+	public async doSendMessage(liveChatUser: { name: string; email: string }, isOffline = true): Promise<void> {
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/page-objects/omnichannel-livechat.ts:42]:
```suggestion
	public async sendMessage(liveChatUser: { name: string; email: string }, isOffline = true): Promise<void> {
```

File: apps/meteor/tests/e2e/utils/create-aux-chatcontext.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import type { Browser, Page } from '@playwright/test';
+
+import { HomeChannel } from '../page-objects';
+
+/**
+ * createTargetChannel:
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/utils/create-aux-chatcontext.ts:6]: Update this comment

File: apps/meteor/tests/e2e/utils/create-aux-chatcontext.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import type { Browser, Page } from '@playwright/test';
```
Comment (souzaramon) [inline apps/meteor/tests/e2e/utils/create-aux-chatcontext.ts:1]:
```suggestion
import { Browser, Page } from '@playwright/test';
```

**Review by souzaramon — APPROVED** [review]

**Review by souzaramon — APPROVED** [review]

**Review by tassoevan — COMMENTED** [review]
~I think the element selectors are quite fragile... Should we do some iteration with @RocketChat/omnichannel to improve them?~

It's a work for later.

**Review by tassoevan — APPROVED** [review]

**Pull request merged**